sql/moz-fx-data-shared-prod/mozilla_vpn_derived/funnel_ga_to_subscriptions_v1/query.sql

```diff
@@ -83,7 +83,7 @@ subscriptions AS (
   WHERE
     DATE(subscription_start_date) >= '2020-07-01'
     AND product_name = "Mozilla VPN"
-    AND provider LIKE "FxA %"
+    AND provider IN ("Stripe", "Paypal")
```

Comment:
This accurately preserves the existing logic, but obscures the intent (why Stripe and Paypal?), which seems like it would make it less likely this logic will be maintained correctly going forward.  An in-code comment mentioning this is filtering to FxA-based providers could help with that.

Reply:
Also, per @birdsarah's comments [in Slack](https://mozilla.slack.com/archives/C018SNLQP6C/p1642614349014400?thread_ts=1642614305.013900&cid=C018SNLQP6C):
> Can the prefix of FxA go away?
> Can the suffix of IAP go away?
> Both would also apply to Google Play making it FxA Google Play IAP
> And Apple subs will be moved onto FxA this year.

- Should Google Play be included in this filter as well, since FxA apparently applies to Google Play?
- Did FxA always apply to Google Play, or is that a change that happened at some point?
- Do we need to be able to distinguish between FxA vs non-FxA subscriptions in an easier/clearer way?  Otherwise it seems like making that distinction will require knowing which providers are FxA-based (and even more specifically during which time periods).

Reply:
> * Should Google Play be included in this filter as well, since FxA apparently applies to Google Play?

no, this funnel only counts subscriptions that signed up through the web flow, and so does not include any type of IAP subscriptions.

> * Did FxA always apply to Google Play, or is that a change that happened at some point?

it always applied

> * Do we need to be able to distinguish between FxA vs non-FxA subscriptions in an easier/clearer way? Otherwise it seems like making that distinction will require knowing which providers are FxA-based (and even more specifically during which time periods).

no, because everything will be FxA.

Reply:
> An in-code comment mentioning this is filtering to FxA-based providers could help with that.

how's this:

```suggestion
    -- only count subscriptions that can have UTMs matching GA, which currently maps to non-IAP providers
    AND provider NOT IN ("Apple Store", "Google Play")
```

Reply:
Looks good, thanks!